src/AutoMapper.Extensions.EnumMapping/Internal/EnumMappingFeature.cs

```diff
@@ -10,6 +10,8 @@ internal class EnumMappingFeature<TSource, TDestination> : IMappingFeature, IEnu
         where TDestination : struct, Enum
     {
         protected EnumMappingType EnumMappingType = EnumMappingType.Value;
+        protected bool IgnoreCase = true;
```

Comment:
default should be false, so remove " = true"

Reply:
✔️ done